kaylimepy/mineral_mystic
# Add plotting function for model metrics

#### Summary:
This PR addresses issue #2 by adding a calling new function, `plot_training_history`, to improve the model training process. The function allows for dynamic plotting of both accuracy and loss metrics.

#### Changes:

1. **Function Call During Training**: 
   - Added a call to `plot_training_history` within the training sequence.
   - Plots are now generated automatically during training.

2. **Commit Validation Plots**: 
   - Validation plots are committed for future reference.
   - Stored under the `plots/` directory.

3. **Validation Plot File Paths**: 
   - Committed plots can be found at:
     - `plots/crystal_vision_model_loss_plot.png`
     - `plots/crystal_vision_model_accuracy_plot.png`

#### Testing:
1. Run the training script: `python run.py`
2. Check `plots/` for the new plots.
